hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -61,6 +69,15 @@ def lambda_handler(event, context):
 
         perform_label_action(action_message, action_message.action_label)
 
+        ActionEventRecord(
+            content_id=action_message.content_key,
+            time_performed=datetime.datetime.now(),
+            action_label=action_message.action_label.value,
```

Comment:
Maybe blocking? Similar to action the hack for action rules, this shouldn't just be the action label (which is included in the action rule) but the entire Action config object. Similar to action rules, eventually we can have a pointer to an ID + version instead.

Reply:
yeah... now that I understand what you are saying this is a larger refactor and probably a different field on this object entirely. For the purposes of this PR I am going to punt but lets create an issue for it and (based on what the team thinks) likely address it before v0 is 'complete' ...

Reply:
Works for me. Issue created: https://github.com/facebook/ThreatExchange/issues/596